src/__tests__/tree-shaking/tree-shaking.spec.ts

```diff
@@ -0,0 +1,159 @@
+import {
+  NgAisModule,
+  NgAisInstantSearchModule,
+  NgAisToggleModule,
+} from '../../index';
+import { Component } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as glob from 'glob';
```

Comment:
We can use `allowSyntheticDefaultImports` in the future to avoid the import `*`.